### DISCUSSION TITLE
Helm chart for Wallos

### DISCUSSION THREAD

**Discussion by vide** [ellite/Wallos#661] [Show and tell]
Hello! I created an Helm chart for Wallos so I can easily install it in my homelab k8s, since I'm migrating it from a docker-compose installation. You can find it here: https://github.com/vide/wallos-charts

If you want, I could open a PR to add it in the README as an installation method. WDYT?

Thanks!

**Comment by ellite** [comment]
Hi @vide 
Thank you very much.
As in other cases in this project, I don't want to link to other sources that I don't have control of what the content might be in the future.
But I would very much like to fork your chart and create an official Wallos chart. Will give you credits on the Readme, of course.
Please let me know if you're fine with it. 
I don't use k8s so I would not know where to start to create a chart on my own 🙈 

Thanks.

**Reply by vide** [reply]
Hey, no problem, just let me fix it because this version has some issues with storage, I need to push some changes I have only locally

**Reply by ellite** [reply]
Perfect. Thank you so much. 🙌
